README.md

```diff
@@ -1,8 +1,8 @@
-![pylint](https://github.com/QuTech-Delft/QMI/tree/main/.github/badges/pylint.svg)
- ![mypy](https://github.com/QuTech-Delft/QMI/tree/main/.github/badges/mypy.svg)
- ![radon-cc](https://github.com/QuTech-Delft/QMI/tree/main/.github/badges/radon-cc.svg)
- ![radon-mi](https://github.com/QuTech-Delft/QMI/tree/main/.github/badges/radon-mi.svg)
- ![coverage](https://github.com/QuTech-Delft/QMI/tree/main/.github/badges/coverage.svg)
+![pylint](.github/badges/pylint.svg?sanitize=true)
+ ![mypy](.github/badges/mypy.svg)
```

Comment:
Won't these other files also eventually need this "sanitize" option? Or is this still part of the test, regarding to the fact that this is really done only at the pull request level?

Reply:
ah yes, I was trying that out, but forgot to remove it at the end. Eventually using relative paths is what made it work and not sanitize. I'll remove the sanitize option